packages/flight-icons/symbol-js/carbon/accessibility.js

```diff
@@ -0,0 +1,3 @@
+export default function (id) {
```

Comment:
What do you think about `.gitignore`ing these build artifacts and just making them something that exists as part of the publish pipeline? (sorry if that's already the plan!)